src/LM_alg.jl

```diff
@@ -54,6 +54,7 @@ function LM(
   verbose = options.verbose
   maxIter = options.maxIter
   maxTime = options.maxTime
+  σmin = options.σmin
```

Comment:
We need to be sure that σ0≥ σmin.

Reply:
I think this is actually a bit later where sigmak is defined but yes good catch.